### PR TITLE
[Feature] #7 Min requirements for landing page

### PR DESCRIPTION
**Pull request by brodia10**
Designed/built the min requirements for #7.

1. Display current game rooms
2. Join a game room
3. Create a new game room

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+        <tr>
+          <td>1</td>
+          <td>2450</td>
+          <td>Easy Room</td>
+          <td>Brook</td>
+          <td>Mike</td>
+          <td>10-5</td>
+          <td><button id="join-btn" type="button" class="btn btn-primary">Join +</button></td>
+        </tr>
+      </tbody>
```
Comment (mvarga6) [inline index.html]: Table looks great!  For the purpose of this page being a placeholder until we actually hook it to the API, let's put only a few items in the list.  I think the important thing show are the 3 different cases we should expect ( we can just make a row for each).
- A room with 2 players and a game underway
- A room with 1 players and is available to join.
- A room with 0 players and is available to join.

Displaying what these 3 cases look like is the priority of the mock data.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,552 @@
...
+              <label for="exampleFormControlInput1" style="color: black;">Room Name</label>
+              <input type="email" class="form-control" id="exampleFormControlInput1"
+                placeholder="your awesome room name">
+            </div>
+            <div class="form-group">
+              <label for="exampleFormControlInput1" style="color: black;">Created by</label>
+              <input type="email" class="form-control" id="exampleFormControlInput1" placeholder="your name">
+            </div>
+            <div class="form-group">
+              <label for="exampleFormControlTextarea1" style="color: black;">Description</label>
```
Comment (mvarga6) [inline index.html]: First off, the modal looks great!  I don't believe we're going to need a field "description" though.  Players can expected that each room will be a game of X-pong.  For the purposes of this placeholder, I think it'll be best to have only fields we _know_ will be utilized immediately.

**Review by mvarga6 — CHANGES_REQUESTED** [review]
This looks really great Brook!  I suggested a few adjustments but then it's good to go!

File: index.html (hunk 2 above)
Comment (brodia10) [reply index.html]: Thanks! That makes sense, appreciate the feedback!

File: index.html (hunk 1 above)
Comment (brodia10) [reply index.html]: Thanks, sounds good!

**Review by mvarga6 — APPROVED** [review]

**Pull request merged**